pages/accountLists/[accountListId]/reports/goalCalculator/index.page.tsx

```diff
@@ -53,13 +49,16 @@ const GoalCalculatorContent: React.FC<GoalCalculatorContentProps> = ({
   designationAccounts,
   setDesignationAccounts,
 }) => {
-  const { rightPanelContent, closeRightPanel } = useGoalCalculator();
+  const { rightPanelContent, closeRightPanel, currentStep } =
+    useGoalCalculator();
   const { t } = useTranslation();
 
   const rightPanel = (
     <>
       <RightPanelHeader>
-        <RightPanelTitle variant="h6">{t('Details')}</RightPanelTitle>
+        <Typography variant="h5" component="div">
+          {currentStep?.title || ''}
+        </Typography>
```

Comment:
This should stay a heading element for better accessibility.

```suggestion
        <Typography variant="h5">
```

Reply:
Is it best to remove the title here and revert to what we had, or to go through and remove the local titles to avoid duplicates? I know you mentioned to wait until I make the change to pass in a title: `openRightPanel(title, component)`